core/src/com/unciv/models/ruleset/unique/Unique.kt

```diff
@@ -57,6 +57,11 @@ class Unique(val text: String, val sourceObjectType: UniqueTarget? = null, val s
                 state.defender != null && state.defender.matchesCategory(condition.params[0])
             UniqueType.ConditionalAttacking -> state.combatAction == CombatAction.Attack
             UniqueType.ConditionalDefending -> state.combatAction == CombatAction.Defend
+            UniqueType.ConditionalVsLargerCiv -> {
+                val yourCities = state.civInfo?.cities?.size ?: 1
```

Comment:
I don't like the implementation. It would return true by default, which is the opposite of what we want.
More like "if state.civInfo null or state.defender null return false"

Reply:
No it returns false by default as 1 < 0 is false. Shouldn't ever trigger as I recall.
Problem is - 'defender' is a little misnomer here, as I expect and from my inspection/testing am almost sure the effect works both on offense and defense (and it should), and the defender is sometimes the aggressor really. But that part of the codebase is not too transparent to me, so if I understood that wrong, tell me.

Reply:
I can't math apparently